src/redis_stack/probabilistic_data_structures.md

```diff
@@ -1,27 +1,39 @@
-In very broad terms probabilistic data structures (PDS) allow us to get to a "close enough" result in a much shorter time and by using significantly less memory.
+This tutorial will demonstrate Redis Stack's probabilistic data structure capabilities using the bike shop use case.
 
-Redis Stack supports 4 of the most famous PDS:
-- Bloom filters
-- Cuckoo filters
-- Count-Min Sketch
+Redis Stack supports the following probabilistic data structures:
+
+- Bloom filter
+- Cuckoo filter
+- Count-min sketch
 - Top-K
+- t-digest
+
+Probabilistic data structures, in general, provide results that are "close enough" in a much shorter time and by using significantly less memory than other data types such as sets or hashes. Here, you'll learn how to use a Bloom filter.
+
+A Bloom filter allows you to check if an element is present in a set using a very small, fixed-size amount of memory. A query will return one of two possible answers:
```

Comment:
Redis implements a scalable Bloom filter. See the EXPANSION parameter in BF.RESERVE. "fixed-size amount of memory" is wrong.

Reply:
Done.